Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -1357,7 +1360,7 @@ WarpXParticleContainer::GetChargeDensity (int lev, bool local)
  * \param lev         Level of box that contains particles
  */
 void
-WarpXParticleContainer::DepositTemperature (amrex::MultiFab* temperature, const int lev)
+WarpXParticleContainer::DepositIsotropicNGPTemperature (amrex::MultiFab* temperature, const int lev)
```

Comment:
This is maybe a bit nit-picky, but I would say "total" or "average" temperature is better than "isotropic", since isotropic, to me, implies the temperature **is** the same in all components, rather than it being a single value collecting the temperature in all directions. Certainly not a big deal though.